tests/parser/2.19.yaml

```diff
@@ -1,5 +1,19 @@
-canonical: 12345
-decimal: +12,345
-sexagesimal: 3:25:45
-octal: 014
-hexadecimal: 0xC
+base2: 0b11010100001011
+base8: 032413
```

Comment:
Can we test some of the awkward cases?

e.g. 038123 (i.e. it should be a string(?))

i.e. what are the edge cases that these rules might fail in. The rules have become much more complex, with significantly more complex regexes, so there are quite a lot more edge cases to check.

Reply:
There should now be some string cases later in the file.